documentation/topics/expressions.md

```diff
@@ -108,7 +108,7 @@ The filter refers to a *single post/comment/tag combination*. So in english, thi
 
 ```elixir
 def has_comment_with_more_points_than(query, score) do
-  Ash.Query.filter(Post, comments.points > 10)
+  Ash.Query.filter(Post, comments.points > score)
```

Comment:
We'll need to use `^score` to refer to the variables.

Reply:
My apologies. Shall submit another PR.